### ISSUE TITLE
Extraneous newlines prepended to message bodies

### ISSUE THREAD

**Issue by Kodiologist** [exteditor/exteditor#61]
When I open a new message, type the letter "a" in the body, and repeatedly invoke the external editor (Emacs) and send the message back to Thunderbird, a newline is added to the beginning of the message body with each attempt at a round trip, except the first time.

I'm using exteditor 2.0.3 and Thunderbird 68.4.1 on Ubuntu 19.10 with the editor command `emacsclient`.

**Comment by JontAlle** [comment]
how did you install it? I have exactly the same version of thunderbird 68 and ubuntu 19.10, but exteditor 2.04 wont install because its not signed.

Did you get a signed version somewhere?

**Comment by Kodiologist** [comment]
No, I enabled installing unsigned add-ons in Thunderbird. I forget how; it was a while ago.
